.ci/magic-modules/inspec-integration.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# TODO make this work
+# Service account credentials for GCP to allow terraform to work
+export GOOGLE_CREDENTIALS_FILE="/tmp/google-account.json"
+# Setup GOPATH
+export GOPATH=${PWD}/go
+
+# CI sets the contents of our json account secret in our environment; dump it
+# to disk for use in tests.
+echo "${GOOGLE_JSON_ACCOUNT}" > /tmp/google-account.json
+
+git clone https://github.com/slevenick/inspec-gcp.git
+pushd inspec-gcp/test/integration
+
+# Generate tfvars
+pushd attributes
+ruby compile_vars.rb > terraform.tfvars
+mv terraform.tfvars ../terraform
+popd
+
+# Run terraform
+pushd terraform
+wget https://releases.hashicorp.com/terraform/0.11.10/terraform_0.11.10_linux_amd64.zip
+apk add unzip
```

Comment:
I think we've been using apt-get instead of apk.

(I'm looking through the Dockerfiles for go-ruby-python and they reference apt-get. The Dockerfiles, which are hopefully up to date, are in `.ci/containers/*/Dockerfile`)

Reply:
Yeah, fixed.